crates/bevy_xpbd_3d/examples/chain_3d.rs

```diff
@@ -95,3 +95,47 @@ fn movement(
         linear_velocity.0 *= 0.9;
     }
 }
+
+fn ui(mut commands: Commands) {
+    commands.spawn(Camera2dBundle {
+        camera: Camera {
+            order: -1,
+            ..default()
+        },
+        ..default()
+    });
+    commands
+        .spawn(NodeBundle {
+            style: Style {
+                width: Val::Percent(100.),
+                ..default()
+            },
+            background_color: Color::rgba(0.15, 0.15, 0.15, 0.0).into(),
+            ..default()
+        })
+        .with_children(|parent| {
+            // text
+            parent.spawn((
+                TextBundle::from_section(
+                    "Use Arrow Keys to Move The Chain",
+                    TextStyle {
+                        font_size: 20.0,
+                        color: Color::WHITE,
+                        ..default()
+                    },
+                )
+                .with_style(Style {
+                    margin: UiRect {
+                        left: Val::Percent(0.5),
+                        top: Val::Percent(2.0),
```

Comment:
This makes the label overlap the FPS counter when the window width is small:

![Text overlap](https://github.com/Jondolf/bevy_xpbd/assets/57632562/9c7e751d-eb9a-4541-a8c9-3d44e4821cf0)

I haven't used Bevy UI much so I'm not sure what the best practises are, but using pixels for the offsets makes the label consistent with the FPS counter:

```suggestion
                        left: Val::Px(5.0),
                        top: Val::Px(30.0),
```